apps/E2E/src/Avatar/specs/Avatar.spec.win.ts

```diff
@@ -34,30 +29,47 @@ describe('Avatar Accessibility Testing', () => {
     await AvatarPageObject.scrollToTestElement();
   });
 
-  it('Validate accessibilityLabel', async () => {
-    await expect(await AvatarPageObject.getPrimaryComponentAttribute(ACCESSIBILITY_LABEL_ATTR)).toEqual(AVATAR_ACCESSIBILITY_LABEL);
+  it('Set accessibilityLabel -> Validate set accessibilityLabel is correct', async () => {
+    await expect(
+      await AvatarPageObject.compareAttribute(AvatarPageObject._primaryComponent, Attribute.AccessibilityLabel, AVATAR_ACCESSIBILITY_LABEL),
+    ).toBeTrue();
+
     await expect(await AvatarPageObject.didAssertPopup()).toBeFalsy(AvatarPageObject.ERRORMESSAGE_ASSERT);
   });
 
-  it('Validate accessibilityLabel from `name` prop', async () => {
-    await expect(await AvatarPageObject.getSecondaryComponentAttribute(ACCESSIBILITY_LABEL_ATTR)).toEqual(
-      AVATAR_ACCESSIBILITY_LABEL_BY_NAME,
-    );
+  it("Set 'name' prop -> Validate accessibilityLabel is correct", async () => {
```

Comment:
What does "correct" mean here? Most people probably don't know that if we only set the 'name' prop and not the accessibilityLabel, then accessibilityLabel defaults to the 'name' prop.

"Correct" doesn't add any value, we should add a statement about what _makes_ it correct:

"Set 'name' prop without setting 'accessibilityLabel' prop -> 'accessibilityLabel' should default to 'name'"